lib/Intlc/Linter.hs

```diff
@@ -1,10 +1,13 @@
 module Intlc.Linter where
+import           Data.Text (all)
 
+import           Data.Char (isAscii)
 import           Intlc.ICU
 import           Prelude   hiding (Type)
 
 data LintingError
   = TooManyInterpolations
+  | EmojiDetected
```

Comment:
As you said on Slack it'd be great to keep track of and print which specific characters tripped the rule.